pkgs/coverage/bin/test_with_coverage.dart

```diff
@@ -160,6 +173,29 @@ ${parser.usage}
     );
   }
 
+  double? failUnder;
```

Comment:
This is duplicated logic with the formatter. Maybe remove it and pass through the flag string unparsed? The formatter step will take care of parsing and validation.